app/cdap/components/ConnectionList/Components/ConnectionTabs/__test__/ConnectionTabs.test.tsx

```diff
@@ -18,19 +18,19 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import ConnectionsTabs from '../index';
 
-const tabsTestData = [{ showTabs: true }];
+const tabsTestData = {data:[], showTabs: true ,selectedTab:"",isSearching:true};
```

Comment:
why do we need this test case updates?
plz let me know

Reply:
Component was throwing an error, hence had to update the props for the same.